core/components/css-utilities/Miscellaneous/Miscellaneous.story.tsx

```diff
@@ -24,20 +24,631 @@ export const miscellaneous = () => {
+            <div className="mb-2 color-accent3-dark font-weight-bold">filter-blur-sm</div>
+            <Card className="p-3 bg-secondary-lightest shadow-s">
+              <img
+                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAwIiBoZWlnaHQ9IjIwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KIDxnPgogIDxyZWN0IHdpZHRoPSIzMDAiIGhlaWdodD0iMjAwIiBmaWxsPSIjZGRkZGRkIi8+CiAgPHJlY3QgeD0iNTAiIHk9IjUwIiB3aWR0aD0iMjAwIiBoZWlnaHQ9IjEwMCIgZmlsbD0iIzU1NTU1NSIvPgogIDx0ZXh0IHg9IjUwJSIgeT0iNTAlIiBmb250LWZhbWlseT0iQXJpYWwiIGZvbnQtc2l6ZT0iMThweCIgZmlsbD0id2hpdGUiIHRleHQtYW5jaG9yPSJtaWRkbGUiIGRvbWluYW50LWJhc2VsaW5lPSJtaWRkbGUiPkV4YW1wbGUgSW1hZ2U8L3RleHQ+CiA8L2c+Cjwvc3ZnPg=="
+                alt="Blur SM"
+                className="filter-blur-sm w-100 vh-15 rounded-10 border border-accent3"
+              />
+            </Card>
+            <Text appearance="subtle" size="small" className="mt-2 color-accent3-dark">
+              Small blur effect (4px)
+            </Text>
+          </div>
+          <div className="w-25 text-center mb-3">
+            <div className="mb-2 color-success-dark font-weight-bold">filter-blur</div>
+            <Card className="p-3 bg-secondary-lightest shadow-s">
+              <img
+                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAwIiBoZWlnaHQ9IjIwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KIDxnPgogIDxyZWN0IHdpZHRoPSIzMDAiIGhlaWdodD0iMjAwIiBmaWxsPSIjZGRkZGRkIi8+CiAgPHJlY3QgeD0iNTAiIHk9IjUwIiB3aWR0aD0iMjAwIiBoZWlnaHQ9IjEwMCIgZmlsbD0iIzU1NTU1NSIvPgogIDx0ZXh0IHg9IjUwJSIgeT0iNTAlIiBmb250LWZhbWlseT0iQXJpYWwiIGZvbnQtc2l6ZT0iMThweCIgZmlsbD0id2hpdGUiIHRleHQtYW5jaG9yPSJtaWRkbGUiIGRvbWluYW50LWJhc2VsaW5lPSJtaWRkbGUiPkV4YW1wbGUgSW1hZ2U8L3RleHQ+CiA8L2c+Cjwvc3ZnPg=="
+                alt="Blur"
+                className="filter-blur w-100 vh-15 rounded-10 border border-success"
+              />
+            </Card>
+            <Text appearance="subtle" size="small" className="mt-2 color-success-dark">
+              Medium blur effect (8px)
+            </Text>
+          </div>
+        </div>
+      </div>
+
+      {/* List style utilities example */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          List Styles
+        </Heading>
+        <div className="d-flex flex-wrap gap-4">
+          <div className="w-25 mb-3">
+            <Card className="p-4 bg-secondary-lightest shadow-s">
+              <Heading appearance="subtle" size="s" className="color-primary-dark mb-2">
+                list-style-disc
+              </Heading>
+              <div className="bg-white p-3 rounded-10 border border-primary">
+                <ul className="list-style-disc ml-4">
+                  <li>Item 1</li>
+                  <li>Item 2</li>
+                  <li>Item 3</li>
+                </ul>
+              </div>
+            </Card>
+          </div>
+          <div className="w-25 mb-3">
+            <Card className="p-4 bg-secondary-lightest shadow-s">
```

Comment:
`shadow-s` as a classname does not exist in our design system. please remove it from everywhere.